fbw-a32nx/src/wasm/systems/a320_systems_wasm/src/lib.rs

```diff
@@ -196,6 +196,8 @@ async fn systems(mut gauge: msfs::Gauge) -> Result<(), Box<dyn Error>> {
     .provides_aircraft_variable("AIRSPEED MACH", "Mach", 0)?
     .provides_aircraft_variable("AIRSPEED TRUE", "Knots", 0)?
     .provides_aircraft_variable("AMBIENT DENSITY", "Slugs per cubic feet", 0)?
+    .provides_aircraft_variable("AMBIENT IN CLOUD", "Bool", 0)?
+    .provides_aircraft_variable("AMBIENT PRECIP RATE", "millimeters of water", 0)?
```

Comment:
Do these not need to be part of `a380_systems_wasm/src/lib.rs` as well?

Reply:
Pretty sure they are !?

Reply:
Ah, don't ask me what I was looking at, found them now, sorry!